src/video/video_source_gst.rs

```diff
@@ -10,6 +10,7 @@ pub enum VideoSourceGstType {
     // TODO: local should have a pipeline also
     Local(VideoSourceLocal),
     Fake(String),
+    Redirect(),
```

Comment:
Since we want to redirect multiple videos, we may want to have a string (for name) to have a identifier for it